draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1068,6 +1108,44 @@ Disclosures:
 
 {{examples/w3c-vc/disclosures.md}}
 
+## Example 4b - W3C Verifiable Credentials Data Model v2.0, using JSON-LD
+
+This example illustrates how to use the artifacts defined in this specification to secure a payload
+that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
+and uses JSON-LD. The example uses a content type `credential+ld+json` defined in [@VC_DATA_v2.0], Section 6.3 in a `cty` JOSE Header value.
+
+SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+
+A Combined Format for Presentation with a Holder Binding JWT would be equivalent to a Holder-signed
+Verifiable Presentation as defined in [@VC_DATA_v2.0].
+
+In this example, Holder Binding is applied and a Combined Format for Presentation is signed
+using a raw public key passed in a `cnf` Claim in the SD-JWT.
+
+Below is a non-normative example of an SD-JWT represented as a W3C VC using JSON-LD.
+
+The following data will be used in this example:
+
+<{{examples/jsonld/user_claims.json}}
+
+The payload of a corresponding SD-JWT looks as follows:
+
+<{{examples/jsonld/sd_jwt_payload.json}}
+
+Disclosures:
+
+{{examples/jsonld/disclosures.md}}
+
+A Combined Format for Presentation for the SD-JWT that discloses only discloses `type`, `medicinalProductName`, `atcCode` of the vaccine,
+`type` of the `recipient`, `type`, `order` and `dateOfVaccination`,
+and has a Holder Binding JWT could look as follows:
```

Comment:
This reads a bit awkward, I think it would be nice to split the claims disclosure from the confirmation binding, but leave both as traits of the example.